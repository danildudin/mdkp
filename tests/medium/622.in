30 6
949 102 498 373 862 1 782 58 384 651 845 263 191 373 29 426 637 224 941 948 82 376 627 786 754 452 872 592 705 740 
526 926 639 145 365 790 274 120 401 655 144 97 696 843 758 969 277 29 683 956 199 759 179 940 699 59 558 434 24 157 6550
493 29 291 66 199 883 676 147 183 802 782 351 69 154 47 151 779 393 685 386 664 679 903 961 259 105 295 129 278 848 5447
487 355 986 752 962 348 809 792 435 393 84 680 153 607 875 703 357 917 588 787 884 627 262 189 258 800 767 556 494 945 9197
718 335 505 227 117 458 752 698 116 369 544 149 433 897 143 892 505 304 472 258 128 465 972 346 262 90 114 518 86 993 5069
126 819 131 980 261 907 121 345 407 136 726 566 572 420 943 980 644 147 892 798 212 533 71 23 84 606 142 422 794 461 9568
147 855 547 187 643 361 316 891 131 177 412 167 100 501 382 997 621 591 466 873 336 65 150 290 76 50 646 279 464 559 7192
