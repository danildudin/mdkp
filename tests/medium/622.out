10224
14
0 2 3 4 6 9 10 18 19 24 25 26 27 28 
